### DISCUSSION TITLE
V Refactoring the compiler, when to start

### DISCUSSION THREAD

**Discussion by Jengro777** [vlang/v#19596] [Questions and Answers] [answered]
I want to know if V has any plans to refactor the compiler, this is important to everyone

**Comment by larpon** [comment, marked as answer]
https://github.com/joe-conigliaro/tinyv

**Comment by JalonSolov** [comment]
Why is it important to everyone?  It will benefit everyone, but most people will never realize it's been done.

**Reply by Jengro777** [reply]
Has the compiler been refactored?
When will this issue  **(https://github.com/vlang/v/issues/19228)**  be fixed?
